feat(nestjs): Update nestjs error monitoring setup (#11141)
getsentry/sentry-docs: platform-includes/getting-started-use/javascript.nestjs.mdx

export class AppModule {}
```

In case you are using a global catch-all exception filter (which is either a filter registered with

> `app.useGlobalFilters()` or a filter registered in your app module providers annotated with an empty `@Catch()` decorator),

Reviewer: ```suggestion
`app.useGlobalFilters()` or a filter registered in your app module providers annotated with a `@Catch()` decorator without arguments),
```

> add a `@WithSentry()` decorator to the `catch()` method of this global error filter. This decorator will report all
> unexpected errors that are received by your global error filter to Sentry:

Reviewer: ```suggestion
add a `@WithSentry()` decorator to the `catch()` method of this global error filter. This decorator will report all
unexpected errors to Sentry that are received by your global error filter:
```
